Sources/SpeziOmron/SpeziOmron.docc/SpeziOmron.md

```diff
@@ -0,0 +1,42 @@
+# ``SpeziOmron``
+
+<!--@START_MENU_TOKEN@-->Summary<!--@END_MENU_TOKEN@-->
+
+<!--
+#
+# This source file is part of the Stanford SpeziDevices open source project
+#
+# SPDX-FileCopyrightText: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
+#
+# SPDX-License-Identifier: MIT
+#
+-->
+
+## Overview
+
+<!--@START_MENU_TOKEN@-->Text<!--@END_MENU_TOKEN@-->
```

Comment:
Nice work with the package target there; would be amazing if we can support this with some cool examples and screenshots here in the landing page.